nrf_security/CMakeLists.txt

```diff
@@ -1,53 +1,165 @@
 #
-# Copyright (c) 2019 Nordic Semiconductor
+# Copyright (c) 2019-2021 Nordic Semiconductor
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 set(NRF_SECURITY_ROOT ${CMAKE_CURRENT_LIST_DIR})
+set(mbedcrypto_target mbedcrypto)
 
 include(${CMAKE_CURRENT_LIST_DIR}/cmake/nrf_security_debug.cmake)
-if(CONFIG_GENERATE_MBEDTLS_CFG_FILE)
-  include(${CMAKE_CURRENT_LIST_DIR}/cmake/kconfig_mbedtls_configure.cmake)
-endif()
 
-include(${CMAKE_CURRENT_LIST_DIR}/cmake/symbol_rename.cmake)
-include(${CMAKE_CURRENT_LIST_DIR}/cmake/symbol_strip.cmake)
-include(${CMAKE_CURRENT_LIST_DIR}/cmake/combine_archives.cmake)
+if(CONFIG_BUILD_WITH_TFM)
+  nrf_security_debug("TFM is enabled, nRF Security will build as part of TF-M")
```

Comment:
```suggestion
  nrf_security_debug("TF-M is enabled, nRF Security will build as part of TF-M")
```

Reply:
Will be fixed, will have a runthrough and remove the usages of the misnomer TFM

Reply:
Fixed